packages/design-system/src/styles/components/_Table.scss

```diff
@@ -40,10 +46,19 @@
   }
 }
 
-.ds-c-table__header--label span {
-  display: block;
-  text-align: center;
-  text-transform: uppercase;
+@mixin ds-c-table--striped-style {
+  thead {
+    th,
+    tr {
+      background-color: $table-stripe-header-background-color;
+    }
+  }
+
+  tbody {
+    tr:nth-child(even) {
+      background-color: $table-stripe-background-color;
+    }
+  }
 }
 
 @mixin ds-c-table--stacked {
```

Comment:
Why is this a mixin instead of a normal class?

Reply:
what do you think if i change it to extend as called in 2 places?

Reply:
Ah right, I forgot. Same feedback as above then, no need for the ds class naming convention, just call it `table-stacked` or `table-stacked-style`